lib/resource-mapper.js

```diff
@@ -149,6 +147,25 @@ class ResourceMapper {
     return { path, contentType: contentType || this._defaultContentType }
   }
 
+  // encode/decode path except slash/%encodedSlash/double%encodedSlash
+  _exceptSlash () { return /(\/|%2f|%2F|%252F|%252f)/g }
+
+  _encodePath (pathname) {
+    const pathArray = pathname.split(this._exceptSlash())
+    pathArray.forEach((el, i) => {
+      if (i % 2 === 0) pathArray[i] = encodeURIComponent(el)
```

Comment:
Might want to normalize path separators to lowercase or uppercase, such that they are mapped to the same filename. (see https://github.com/CommunitySolidServer/CommunitySolidServer/pull/1273)